crypto/nrf_cc310_platform/include/nrf_cc3xx_platform_sha256.h

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/**@file
+ * @defgroup nrf_cc3xx_platform_sha256 nrf_cc3xx_platform SHA256 APIs
+ * @ingroup nrf_cc3xx_platform
+ * @{
+ * @brief The nrf_cc3xx_platform_sha256 APIs provide functions for
+ *        hashing data using SHA256 algorithm.
+ */
+
+#include <stddef.h>
```

Comment:
Not worth fixing.

But FYI, includes should be below the header guard.

Reply:
Will be fixed